docs/adguard-home/environment.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Environment variables
+sidebar_position: 5
+---
+
+This page lists the supported environment variables and their descriptions.
+
+### `ADGUARD_HOME_DEFAULT_WEB_PORT`
```

Comment:
1. It's better to use the second level (`##`), I think.
2. Add an explicit id with `{#ADGUARD_HOME_DEFAULT_WEB_PORT}`.  See, for example, `./docs/dns-client/configuration.md`.